RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -500,7 +500,7 @@ class RNUnifiedContacts: NSObject {
 
     func contactContainsText( _ cNContact: CNContact, searchText: String ) -> Bool {
         let searchText   = searchText.lowercased();
-        let textToSearch = cNContact.givenName.lowercased() + " " + cNContact.familyName.lowercased()
+        let textToSearch = cNContact.givenName.lowercased() + " " + cNContact.familyName.lowercased() + cNContact.nickname.lowercased()
```

Comment:
I _think_ you need to add an empty space (`" "`) here between the familyName and nickname.